src/analyzer.ts

```diff
@@ -57,7 +57,7 @@ export class NoKnownParserError extends Error {};
 export class Analyzer {
   private _parsers = new Map<string, Parser<ParsedDocument<any, any>>>([
     ['html', new HtmlParser()],
-    ['js', new JavaScriptParser()],
+    ['js', new JavaScriptParser({sourceType: 'script'})],
     ['css', new CssParser()],
     ['json', new JsonParser()],
```

Comment:
You know, file extension isn't the only way we need to grab parsers.

Like, if something has been html-imported, then it needs to be handled as html. If something has been grabbed with `<script src='foo.bar>` it needs to be treated as javascript. And if something has been imported as an ES6 module it needs to be handled as a JS module.